File: test_cadastro_rendimentos.java
package tst;

import static org.junit.Assert.*;

import org.junit.Before;
import org.junit.Test;

import app.IRPF;

public class CadastroRendimentosTest {

    private IRPF irpf;

    @Before
    public void setup() {
        irpf = new IRPF();
    }

    @Test
    public void testCadastrarUmRendimento() {
        irpf.cadastrarRendimento(5000f);
        assertEquals(5000f, irpf.getTotalRendimentos(), 0);
    }

    @Test
    public void testCadastrarDoisRendimentosIguais() {
        irpf.cadastrarRendimento(6000f);
        irpf.cadastrarRendimento(6000f);
        assertEquals(12000f, irpf.getTotalRendimentos(), 0);
    }

    @Test
    public void testCadastrarDoisRendimentosDiferentes() {
        irpf.cadastrarRendimento(5000f);
        irpf.cadastrarRendimento(6000f);
        assertEquals(11000f, irpf.getTotalRendimentos(), 0);
    }
}

File: test_cadastro_rendimentos.py
import unittest
from irpf import IRPF

class CadastroRendimentosTest(unittest.TestCase):

    def setUp(self):
        self.irpf = IRPF()

    def test_cadastrar_um_rendimento(self):
        self.irpf.cadastrar_rendimento(5000)
        self.assertEqual(5000, self.irpf.get_total_rendimentos())

    def test_cadastrar_dois_rendimentos_iguais(self):
        self.irpf.cadastrar_rendimento(6000)
        self.irpf.cadastrar_rendimento(6000)
        self.assertEqual(12000, self.irpf.get_total_rendimentos())

    def test_cadastrar_dois_rendimentos_diferentes(self):
        self.irpf.cadastrar_rendimento(5000)
        self.irpf.cadastrar_rendimento(6000)
        self.assertEqual(11000, self.irpf.get_total_rendimentos())
